src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+min_read: 5
+date: 2020-02-15
+---
+
+### Wat is Phaser?
+Kort gezegd is [Phaser](https://phaser.io/) een gameframework waarmee je HTML-5 games kan maken. Deze games kunnen gespeeld worden op desktop en mobiel. Phaser kan gratis gedownload worden via https://phaser.io/ en is ontwikkeld door Photon Storm in 2013.
+
+Om gebruik te maken van Phaser is het handig als je al enige kennis hebt van HTML en JavaScript. Phaser is namelijk een JavaScript library, wat dus betekent dat je de game volledig hierin bouwt. Phaser maakt het mogelijk voor beginners en gevorderden om een eigen interactieve en leuke game te maken. Verder heeft het veel verschillende functies, zelfs te veel om allemaal te benoemen. Dit is ook een van de redenen waarom het zo geliefd is bij game ontwikkelaars. 
+
+### Opzetten van een Phaser game
+Het begint allemaal bij je HTML bestand waar je Phaser aanroept in je head, met de volgende code: 
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/phaser-ce/2.12.0/phaser.min.js"></script>
+```
+Na dit gedaan te hebben maak je een JavaScript bestand aan en roep je de game aan met:
+```
+window.onload = function(){
+    var game = new Phaser.Game(800 (width), 600 (height), Phaser.AUTO, "", {
```

Comment:
Ik zou _(width)_ en _(height)_ misschien weg halen uit de code. 
Mocht iemand het willen testen en de code kopieren, werkt de code niet (lijkt mij). Ik zou het denk ik toellichten in de tekst welke parameters de functie nodig heeft.

Reply:
Heb je gelijk in, dus heb het veranderd.